testgrid/config.yaml

```diff
@@ -7634,6 +7653,31 @@ dashboards:
     test_group_name: release-openshift-origin-installer-e2e-aws-4.0
     base_options: width=10
 
+#Docker conformance dashboard
+- name: sig-node-docker
+  dashboard_tab:
+  - name: ce18.09-node.e2e-ubuntu-k8s1.13
+    test_group_name: ce18.09-ubuntu16.04-k8s1.13.x-conformance
+    description: k8s1.13.x conformance test results for docker ce-18.09 on ubuntu16.04
```

Comment:
node conformance != conformance. which is this?

Reply:
it is node conformance , that is why it is added under sig-node group thx